__tests__/app.test.js

```diff
@@ -37,12 +37,42 @@ describe('/api', () => {
     return request(app)
     .get('/api')
     .then((response) => {
-      console.log(response.body)
       expect(response.body.endpoints).toEqual(endpoints)
     })
   })
 })
 
+describe('/api/articles/:article_id', () => {
+  test('GET 200: Responds with an article when we query by id', () => {
+  return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({ body }) => {
+      const { article } = body;
+      console.log(article)
+      expect(article).toMatchObject({
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 100,
+        article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+    });
```

Comment:
This is a very good test, two points of feedback from me:
- remove console logs when you're finished developing ( line 52 🧹 )
- Because this has been retrieved from the database we would also expect the object to have a property of `article_id: 1`